### PR TITLE
BACK-18: Backend Setup

### PR DESCRIPTION
**Pull request by DogukanGun**
## Description of the issue
Setup the node project.

## How I implemented
npm start

### CODE REVIEW

File: controller/example.controller.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+
+    THIS IS AN EXAMPLE FOR CONTROLLER. PLEASE DONT DELETE THIS FILE FOR 2 WEEKS
+
+*/    
+
+
+const books:any[] = [];
```
Comment (pczern) [inline controller/example.controller.ts:8]: space books: any[]
what is books?

Comment (pczern) [reply controller/example.controller.ts:8]: don't use any

**Comment by pczern** [comment]
Create an own folder for `queries`

**Comment by pczern** [comment]
abstract graphql from the actual router, by bundling all graphql logic in a separate folder maybe called `graphql`
* graphql/schema
* graphql/queries
* graphql/resolvers

**Comment by pczern** [comment]
bin/www???????? I think in JS world it's generally accepted to have a `server.js`, we don't want to adhere to linux standards?

File: controller/example.controller.ts (hunk 1 above)
Comment (DogukanGun) [reply controller/example.controller.ts:8]: This is just an example to show how they should create endpoints

Comment (pczern) [inline controller/example.controller.ts:8]: add a space before types

File: controller/example.controller.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const getBooks = () => books;
+
+const addBook = (title:string, author:string) => {
```
Comment (pczern) [inline controller/example.controller.ts:12]: add space before types

File: package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "backend",
```
Comment (pczern) [inline package.json:2]: add "license": "UNLICENSED"

**Review by pczern — CHANGES_REQUESTED** [review]
* Change PR title
* Add PR description
* Add PR How I implemented
* Abstract graphql from the actual router, by bundling all graphql logic in a separate folder maybe called graphql
_graphql/schema_
_graphql/queries_
_graphql/resolvers_
* Create an own folder for `graphql/queries`
* `bin/www` looks like Linux, I think we should adhere to common practice and call it just `server.js`

**Comment by pczern** [comment]
This issue is also related to issue META-18, META-30 and BACK-18.
New PR is available, replacing this PR. Closing this for now.

**Pull request closed without merging**